downstream/titles/onboarding/master.adoc

```diff
@@ -63,7 +49,20 @@ To use Automation Service Catalog as a Lighthouse customer, create a new Red Hat
 The account number for your Red Hat corporate account is required to use Automation Service Catalog, but  is not provided automatically to Red Hat staff when creating a new account.
 ====
 
-== Creating groups and users
+The required entitlements are added and you can now access Automation Service Catalog.
+
+== Groups and users
+
+When configuring Automation Service Catalog for your needs, consider the scope of your user environments and how those users are grouped.
+
+Determine the required level of access for your users based on the following predefined roles in Automation Service Catalog:
+
+* *Organization Administrator*: a Red Hat account user who can manage users and control their access and permissions.
+* *Approval Administrator*: 	An Approval administrator role that grants create, read, update and destroy permissions.
```

Comment:
@bzwei Should we be adding the objects that are involved here, request, stage, action etc.?

Reply:
Approval Administrator role grands create, read, update and destroy permissions to approval process objects, plus permissions for approval approver and approval user roles.

Reply:
Working with UX to provide more descriptive explanations. RBAC docs have full description at present and we will link over to that docs set for any in-depth needs customers may have.